.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: node_js
+node_js: 
+  - 8
+env:
```

Comment:
For the `env` key, any variables listed immediately under it will trigger multiple builds in Travis with each env var being set to a build (see [here](https://travis-ci.org/kdipaolo/Family-Challenge/builds/319759533)). You want to set the Surge env vars as [global variables](https://docs.travis-ci.com/user/environment-variables/#Global-Variables).

Reply:
Got it! That makes sense. Thanks!